vendor/comtac/lpn-km-codec.yaml

```diff
@@ -0,0 +1,112 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: lpn-km.js
```

Comment:
hi @xExecutioner please check your.js file name, the name of the file is lpn_km.js`